.github/workflows/e2e_additional_editor_and_player.yml

```diff
@@ -38,9 +38,13 @@ jobs:
       - name: Run Additional Editor E2E Test
         if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
         run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --suite="additionalEditorFeatures" --prod_env
+        env: 
+          VIDEO_RECORDING_IS_ENABLED: ${{ true }}
```

Comment:
I think it would be better to set it at each step since in the future we will be re-enabling recordings one-by-one for each suite (this would be useful then).